sklearn/utils/stats.py

```diff
@@ -41,13 +43,17 @@ def _weighted_percentile(array, sample_weight, percentile_rank=50, xp=None):
     xp : array_namespace, default=None
         The standard-compatible namespace for `array`. Default: infer.
 
+    symmetrize : bool, default=False
```

Comment:
Nit about this name - could we make it more consistent with how it's named elsewhere? I know this function is private and we ultimately would like to use the scipy quantile so we don't have to maintain our own, but it would still be nice if it was clearer what this method was equivalent to.

I *think* this quantile method is called "averaged_inverted_cdf" in numpy/scipy. And we were calling it `_averaged_weighted_percentile`. I can understand "symmetrize" term but what about "average" or "average_inverted"  etc ?